examples/tutorials/nb_python/ECCV_2020_Navigation.py

```diff
@@ -0,0 +1,1130 @@
+    # default = 45.0
+
+    # @markdown **Navigable area filtering options**:
+    navmesh_settings.filter_low_hanging_obstacles = True  # @param {type:"boolean"}
+    # default = True
+    navmesh_settings.filter_ledge_spans = True  # @param {type:"boolean"}
+    # default = True
+    navmesh_settings.filter_walkable_low_height_spans = True  # @param {type:"boolean"}
+    # default = True
+
+    # @markdown **Detail mesh generation parameters**:
+    # @markdown For more details on the effects
+    navmesh_settings.region_min_size = (
+        20
+    )  # @param {type:"slider", min:0, max:50, step:1}
+    # default = 20
+    navmesh_settings.region_merge_size = (
+        20
+    )  # @param {type:"slider", min:0, max:50, step:1}
+    # default = 20
+    navmesh_settings.edge_max_len = (
+        12.0
+    )  # @param {type:"slider", min:0, max:50, step:1}
+    # default = 12.0
+    navmesh_settings.edge_max_error = (
+        1.3
+    )  # @param {type:"slider", min:0, max:5, step:0.1}
+    # default = 1.3
+    navmesh_settings.verts_per_poly = (
+        6.0
+    )  # @param {type:"slider", min:3, max:6, step:1}
+    # default = 6.0
+    navmesh_settings.detail_sample_dist = (
+        6.0
+    )  # @param {type:"slider", min:0, max:10.0, step:0.1}
+    # default = 6.0
+    navmesh_settings.detail_sample_max_error = (
+        1.0
+    )  # @param {type:"slider", min:0, max:10.0, step:0.1}
+    # default = 1.0
+
+# @markdown ---
+# @markdown You can also include MotionType::STATIC objects as NavMesh obstacles:
+
+include_static_objects = True  # @param {type:"boolean"}
+
+navmesh_success = sim.recompute_navmesh(
+    sim.pathfinder, navmesh_settings, include_static_objects
+)
+
+observations = sim.get_sensor_observations()
+rgb = observations["color_sensor"]
+semantic = observations["semantic_sensor"]
+depth = observations["depth_sensor"]
+
+display_sample(rgb, semantic, depth)
+
+
+# %%
+# @markdown ##Saving the NavMesh
+
+# @markdown An existing NavMesh can be saved with *Pathfinder.save_nav_mesh(filename)*
+if sim.pathfinder.is_loaded:
+    navmesh_save_path = (
+        "/content/habitat-sim/data/test_saving.navmesh"
+    )  # @param {type:"string"}
+    sim.pathfinder.save_nav_mesh(navmesh_save_path)
+    print('Saved NavMesh to "' + navmesh_save_path + '"')
+    sim.pathfinder.load_nav_mesh(navmesh_save_path)
+
+# %%
+# @title Define Simulation and Video Utlities { display-mode: "form" }
+
+import cv2
+import git
+
+repo = git.Repo(".", search_parent_directories=True)
+dir_path = repo.working_tree_dir
+data_path = os.path.join(dir_path, "data")
+# @markdown Optionally configure the save path for video output:
+output_directory = "examples/tutorials/nav_output/"  # @param {type:"string"}
+output_path = os.path.join(dir_path, output_directory)
+if not os.path.exists(output_path):
+    os.mkdir(output_path)
+
+
+def make_video_cv2(observations, prefix="", open_vid=True, fps=60, multi_obs=False):
```

Comment:
This should be in viz_utils now, unless this function is different somehow?

Reply:
We're targeting more work on this soon. 👍

Reply:
Yeah, sorta got held up with finishing up scene import.  We're working on it :)